packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -27,10 +27,12 @@ baseChainId = "8453"
 owner = "0x1Fe50318e5E3165742eDC9c4a15d997bDB935Eb9"
 passivePoolExchangeFeeCollectorOwner = "0x93701b8f4e212B4eC058AF375894DB4B64b4f708"
 lotteryPotExchangeFeeCollectorOwner = "0x1Fe50318e5E3165742eDC9c4a15d997bDB935Eb9"
+rageTradeExchangeFeeCollectorOwner = "0x507c7777837b85ede1e67f5a4554ddd7e58b1f87"
```

Comment:
is this address + account id below provided by rage (or did we just create an account for rage and assigned ownership to their address)?

Reply:
yes, these are the address + account id provided by rage